plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiErrorCode.java

```diff
@@ -28,7 +28,9 @@
     HUDI_MISSING_DATA(3, EXTERNAL),
     HUDI_CANNOT_OPEN_SPLIT(4, EXTERNAL),
     HUDI_UNSUPPORTED_FILE_FORMAT(5, EXTERNAL),
-    HUDI_CURSOR_ERROR(6, EXTERNAL);
+    HUDI_CURSOR_ERROR(6, EXTERNAL),
+
+    HUDI_INVALID_UER_INPUT_INSTANT_TIME_FORMAT(7, EXTERNAL);
```

Comment:
I expect "the config value is wrong" from this naming, but this is used for handling failure when parsing input value from Hudi.